public/app/features/logs/components/LogRow.tsx

```diff
@@ -65,6 +65,8 @@ interface Props extends Themeable2 {
   pinned?: boolean;
   containerRendered?: boolean;
   handleTextSelection?: (e: MouseEvent<HTMLTableRowElement>, row: LogRowModel) => boolean;
+  logRowMenuIconsBefore?: ReactNode[];
+  logRowMenuIconsAfter?: ReactNode[];
```

Comment:
what do you think about adding one new `logRowMenuIcons` object, with children props for `before` and `after`? Might make refactor in the future a bit easier (I'm thinking we might eventually want to add an index, or weight instead of before/after at some point)

Reply:
I'm curious to know what would be the advantage of doing it that way, so let's chat. For context, it's been inspired in these props https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/Input/Input.tsx#L22-L25 which seems to be used in a couple of places already https://github.com/search?q=repo%3Agrafana%2Fgrafana%20addonBefore&type=code

Reply:
No real advantage besides keeping the number of props down, probably not worth spending any time on, especially if we're following this pattern elsewhere in core.